sources/assets/Stride.Core.Assets/AssetRegistry.cs

```diff
@@ -364,7 +439,31 @@ public static IEnumerable<IDataCustomVisitor> GetDataVisitNodes()
             }
         }
 
-        public static bool IsContentType(Type type)
+        /// <summary>
+        /// Can a property of this type hold content
+        /// </summary>
+        /// <remarks>
+        /// You may want to use <see cref="IsExactContentType"/> instead if you're checking if this type is a concrete content type
+        /// </remarks>
+        public static bool CanBeAssignedToContentTypes([Annotations.NotNull] Type type, bool checkIsUrlType)
+        {
+            lock (RegistryLock)
+            {
+                if (checkIsUrlType && (type.IsAssignableTo(typeof(AssetReference)) || UrlReferenceBase.IsUrlReferenceType(type)))
+                    return true;
+
+                return AssignableToContent.ContainsKey(type);
+            }
+        }
+
+        /// <summary>
+        /// Is this type a concrete content type
```

Comment:
Description doesn't really match the method name.

Reply:
Changed it a bit, let me know